components/gitpod-db/src/typeorm/entity/db-layout-data.ts

```diff
@@ -7,13 +7,12 @@
 import { PrimaryColumn, Column, Entity } from "typeorm";
 
 import { LayoutData } from "@gitpod/gitpod-protocol";
-import { TypeORM } from "../typeorm";
 import { Transformer } from "../transformer";
 
 @Entity()
 export class DBLayoutData implements LayoutData {
 
-    @PrimaryColumn(TypeORM.UUID_COLUMN_TYPE)
+    @PrimaryColumn("varchar")
```

Comment:
The original type hat a fixed length. Considering how important the workspace ID is, I think it would make sense to introduce a specific `WORKSPACE_ID_TYPE` type, akin to `UUID_COLUMN_TYPE`.

Otherwise we'll end up with new tables that use `VARCHAR(255)` for workspace IDs in the future.

Also, considering that workspace IDs have roughly the same char length, we would probably see better performance if we used `CHAR` columns (like we have in the past). See e.g. https://dba.stackexchange.com/questions/424/performance-implications-of-mysql-varchar-sizes/1915#1915

Reply:
Good point. We should keep the type (char 36). I'll just define a new column type `TypeORM.WORKSPACE_ID_COLUMN_TYPE` with the same technical type mapping.

Reply:
I don't think migration are needed then.